linera-client/src/persistent/dirty.rs

```diff
@@ -2,11 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #[derive(Clone, Debug, PartialEq, Eq, derive_more::Deref, derive_more::DerefMut)]
-pub struct Dirty(
-    #[deref]
-    #[deref_mut]
-    bool,
-);
+pub struct Dirty(#[deref] bool);
```

Comment:
What happened to `deref_mut`?

Reply:
ups! I think I introduced `deref_mut` intermittently in this PR, then removed it and went too far with this.

Reply:
Reverted.